MdsSupportingClasses/MdsColliveryService.php

```diff
@@ -431,12 +438,12 @@ public function getPrice(array $array, $adjustedTotal, $markup, $fixedPrice)
             $discount = 0;
         }
 
+        $returnedAmount = $result['data'][0]['total'];
+
         if ($this->settings->getValue('include_vat') === 'yes') {
-            $returnedAmount = $result['price']['inc_vat'];
-        } else {
-            $returnedAmount = $result['price']['ex_vat'];
+            $returnedAmount *= 1.15;
```

Comment:
VAT should be determined by a setting 
see /MdsSupportingClasses/MdsColliveryService.php#L434

Reply:
It does The full extent of that section is
`if ($this->settings->getValue('include_vat') === 'yes') {
            $returnedAmount *= 1.15;
        }`

Reply:
Apologies, I see that now.